### PR TITLE
Update homework.md

### PR DESCRIPTION
**Pull request by Sarah-Nahas**
correct answers

# Your Details

- Your Name:Sareh Nahas 
- Your City:London
- Your Slack Name:Sarah Nahas

# Homework Details

- Module:Git and GitHub
- Week:1

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,111 @@
...
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```
Comment (Sharon-Akaka) [inline homework.md:17]: Hi Sarah, this should say "What is the 2 * 3 * 6 * 1 * 10 * 34?"

File: homework.md [hunk 2]
```diff
@@ -4,110 +4,111 @@
...
 
-1322
```
Comment (Sharon-Akaka) [inline homework.md:19]: This is the correct answer of the equation

File: homework.md [hunk 3]
```diff
@@ -4,110 +4,111 @@
...
+1-Remove a file from git without removing it from your file system. ...
+2-Undo local commits. ...
+3-Clean up local commits before pushing. ...
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Iis producing a personal copy of someone else's projectt
```
Comment (Sharon-Akaka) [inline homework.md:60]: Just some typo corrections:

It is producing a personal copy of someone else's project

File: homework.md [hunk 4]
```diff
@@ -4,110 +4,111 @@
...
 
-## Question 10 
+## Question 10
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
```
Comment (Sharon-Akaka) [inline homework.md:64]: Great spelling mistake catch!

File: homework.md [hunk 5]
```diff
@@ -4,110 +4,111 @@
...
 
-To record what changes you made to the code
+is used to save changes to a local repository after staging in Git
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The Top bar there is a word "Help"
```
Comment (bisioolajide) [inline homework.md:114]: Great call on the "Help" button Sarah. I didn't even think of that option. You might also want to include the "Git cheatsheet" and Slack in your answer though.

**Comment by bisioolajide** [comment]
Great job Sarah, just one comment I noticed on Q18 you might want to recheck

File: homework.md (hunk 5 above)
Comment (Sarah-Nahas) [reply homework.md:114]: Thanks, Bisioolajide for pointing that out I will update it.

File: homework.md (hunk 4 above)
Comment (Sarah-Nahas) [reply homework.md:64]: thank you!

File: homework.md (hunk 3 above)
Comment (Sarah-Nahas) [reply homework.md:60]: thanks, Sharon for the corrections. 
Do you know how I can correct these mistakes?

**Comment by DibaKarimi** [comment]
Hi Sarah , well done!
One thing you might want to edit is the title of your pull request by your name.

**Pull request closed without merging**